src/common/IBackend.ts

```diff
@@ -56,4 +56,9 @@ export interface IBackend {
    * Update formData to allow valid POST
    */
   updateFormData(folder: Folder, file: any, formData: any): Promise<any>;
+
+  /**
+   * Upload file using multi-part uploading
+   */
+  multipartUpload(params: any): Promise<any>;
```

Comment:
Since this is the only method we have for uploading files, would it make sense to give it a name that is less specific to aws?  Such as uploadFile()

Reply:
That makes sense for me, I'll make that change, thanks!